src/utils/FindUserById.tsx

```diff
@@ -0,0 +1,6 @@
+import usersFromServer from '../api/users';
+import { User } from '../types/User';
+
+export const findUserById = (id: number): User => {
+  return usersFromServer.find(user => user.id === id)!;
+};
```

Comment:
Don't disable typescript with the `!` operator!!!!
```suggestion
export const findUserById = (id: number): User | null => {
  return usersFromServer.find(user => user.id === id) || null;
};
```